### DISCUSSION TITLE
Passing the WHOLE field with its data to the Vue component with antlers

### DISCUSSION THREAD

**Discussion by michalmowinski** [statamic/cms#7301] [Q&A]
In CMS I've created a field 'replicator_list' with type 'replicator' which contains one set with two fields: 
1) title (text)
2) elements (entries connected with collection)

For example I created two items:
1. 
  title: 'some title 1'
  elements: { element1, element2 }
2.
  title: 'some title 2'
  elements: { element1, element2, element3 }

  where each of chosen element is kind of an object with a lot of data.

in antlers.html I could simply iterate throw replicator_list and pass the elements to the Vue component:

`{{ replicator_list }}`
  `<sample-slider :elements='{{ replicator_list.elements }}'></sample-slider>`
`{{ /replicator_list }}`

But what I need is to pass the WHOLE field 'replicator_list' to the Vue component:

`<sample-slide :replicator_list='{{ replicator_list | to_json }}'></sample-slider>`

and when I display the replicator_list inside sample-slide component I received:
`[
  {
    title: 'some title 1',
    elements: {}
  },
  {
    title: 'some title 2',
    elements: {}
  },
]`

The 'elements' is an empty object but I was expecting a list of elements I added in CMS with its data.

Could you help me solve my problem?

**Comment by jasonvarga** [comment]
Not at the moment.
See #6299, #5866, #6130
